tests/conftest.py

```diff
@@ -249,3 +251,45 @@ def persisted_channel(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="package")
+def grpc_session_creation_kwargs():
+    """Gets the keyword arguments required for creating the gRPC interpreter."""
+    channel = grpc.insecure_channel(f"localhost:31763")
+    grpc_options = GrpcSessionOptions(
+        grpc_channel=channel,
+        session_name="",
+    )
+    grpc_options = GrpcSessionOptions(channel, "")
+    return {"grpc_options": grpc_options}
+
+
+@pytest.fixture(scope="package")
+def library_session_creation_kwargs():
```

Comment:
Naming: nimi-python calls this `session_creation_kwargs` because it's used when creating a `Session` object. In nidaqmx-python, there is no `Session` object, and these will be used to create a lot of different objects, so consider renaming to something more generic like `init_kwargs`.

Reply:
I have renamed the fixtures as per suggestions.